src/PuzzleScraper.py

```diff
@@ -0,0 +1,34 @@
+from bs4 import BeautifulSoup
+import requests
+import re
+
+class problem:
+    def __init__(self, fen, attributes):
+        self.FEN = fen
+        self.ID = attributes[0].text.strip()
+        self.Rating = attributes[1].text.strip()
+        self.Attempts = attributes[2].text.strip()
+        self.PassPercent = attributes[3].text.strip()
+        self.NumMoves = attributes[4].text.strip()
+        self.AvgTime = attributes[5].text.strip()
+        self.PlayAsColor = attributes[6].text.strip()
+#6720 pages in total
+problems=[]
+for pagenum in range(1,4):

```

Comment:
Is it just 4 pages?

Reply:
No, it's that int should be 6721. I'll make a new commit fixing that and the data type for the ID and rating
```suggestion
for pagenum in range(1,6721):

```